README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# Homework 1
+
+## Rusty Roberts
+
+My name is Rusty! I was born in Cleveland, Ohio, the *best* city in the world! You might agree, you might not, but you can't argue with the *emphasis* I've placed on the word *best* to make my claim more compelling. After attending college at the College of Wooster in Wooster, Ohio, I worked as a teachcer in New York City for eight years. I'm now a doctoral student at Booth studying behavioral science. 
+
+### Favorites!
+
+**Foods**
+* Spaghetti
+* Tacos
+* Popcorn
+
+**TV Shows**
```

Comment:
Nice use of lists; you can tab to indent the sub-items and make it more readable